### PR TITLE
Add Matomo (former Piwik) analytics script

### PR DESCRIPTION
**Pull request by Volker-E**
Bug: [T188786](https://phabricator.wikimedia.org/T188786)

### CODE REVIEW

File: design-principles.html [hunk 1]
```diff
@@ -20,6 +20,26 @@
     <link rel="preload" href="fonts/Charter_regular.woff2" as="font" type="font/woff2" crossorigin>
     <link rel="preload" href="fonts/Lato_regular.woff2" as="font" type="font/woff2" crossorigin>
     <script src="js/fonts-loader.js" async></script>
+	<script>
```
Comment (Ladsgroup) [inline design-principles.html]: They are not aligned with the above line

File: index.html [hunk 2]
```diff
@@ -20,6 +20,26 @@
     <link rel="preload" href="fonts/Charter_regular.woff2" as="font" type="font/woff2" crossorigin>
     <link rel="preload" href="fonts/Lato_regular.woff2" as="font" type="font/woff2" crossorigin>
     <script src="js/fonts-loader.js" async></script>
+	<script>
```
Comment (Ladsgroup) [inline index.html]: Same here

**Comment by Ladsgroup** [comment]
I recommend factoring them out into one single js file and just include that instead of copy pasting it everywhere, that makes improvements on the code also easier later.

File: index.html (hunk 2 above)
Comment (Volker-E) [reply index.html]: Oh, well, above three are wrong.

Comment (Volker-E) [reply index.html]: Done in next pushed commit

**Comment by Volker-E** [comment]
@Ladsgroup I agree with putting the code into external file. But if we do so we need to make sure, it's going together with all the right preconditions (like eslinting, minification) and I'd like to ensure the functionality first in this step (if Matomo is getting the data, the performance is ok). 
Bringing in the other packages and build steps seems to much for one PR.

**Pull request merged**